### ISSUE TITLE
 Get 'TypeError: Super expression must either be null or a function' after run build

### ISSUE THREAD

**Issue by trungdinh98** [NadavShaar/react-grid-table#94]
In local, it work correctly, but when run build folder is created after run command 'yarn build' on server, I get error "TypeError: Super expression must either be null or a function" when load page use <GridTable/>

**Comment by NadavShaar** [comment]
Hi are you still facing this issue?

**Comment by RaquelPlaza** [comment]
I am also experiencing the same error, I installed the package and used the example code, build my project and got that error: TypeError: Super expression must either be null or a function

**Comment by VivekTodur** [comment]
No, it still the same. I have tried with the fresh project as well without loading any other module, but error remains same. I am using NodeJS v18.9.1, ReactJS v18.2.0
